core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/host/ClusterHostServiceRunner.java

```diff
@@ -678,6 +680,11 @@ private void decoratePillarWithTags(StackView stack, Map<String, SaltPillarPrope
         }
     }
 
+    private boolean isCloudProviderTypeSupported(StackDto stackDto) {
+        List<String> supportedCloudTypes = Arrays.asList("AWS", "AZURE", "GCP", "NONE");
```

Comment:
this most probably should be a constant, and the values to come from the already existing `CloudPlatform`
also I don't see any unit tests for this